MonoGame.Framework/Input/GamePad.UWP.cs

```diff
@@ -92,6 +96,13 @@ private static GamePadState PlatformGetState(int index, GamePadDeadZone deadZone
                 (state.Buttons.HasFlag(WGI.GamepadButtons.X) ? Buttons.X : 0) |
                 (state.Buttons.HasFlag(WGI.GamepadButtons.Y) ? Buttons.Y : 0) |
                 0;
+
+            // Check triggers
```

Comment:
I am sorry for complaining to much on this PR... anyway last thing, don't write obvious comments. It's like saying:
```
// increase i by 1
i++;
```